### PR TITLE
Manifests for k8s-container-image-promoter prow jobs

### PR DESCRIPTION
**Pull request by dims**
Placeholders for manifests for use with https://github.com/GoogleCloudPlatform/k8s-container-image-promoter

Change-Id: Ic5a0eb9be30e5bd4682faa484436fb0d543235ce

### CODE REVIEW

**Comment by thockin** [comment]
/approve

I'll let @listx check syntax

**Comment by thockin** [comment]
Maybe worth including the googlegroup as a comment for each staging repo

**Comment by dims** [comment]
Already done :) see the first line of the yaml

**Comment by thockin** [comment]
As per zoom, let's move the script from the current "gcr" dir to this root in the same PR.

**Comment by thockin** [comment]
I have almost totally rewritten the script.  Will wait until you git-move it before sending a PR

File: k8s.gcr.io/k8s-staging-csi/manifest.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# google group for gcr.io/k8s-staging-csi is k8s-infra-gcr-staging-csi@googlegroups.com
+registries:
+  src: gcr.io/k8s-staging-csi
+  dest: gcr.io/k8s-staging
+service-account: service-account@kubernetes-public.iam.gserviceaccount.com
+#images:
```
Comment (listx) [inline k8s.gcr.io/k8s-staging-csi/manifest.yaml]: Can you uncomment this line? The `images` key is part of a Go struct so the marshaling logic will break if it is missing IIRC.

File: k8s.gcr.io/k8s-staging-cluster-api/manifest.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# google group for gcr.io/k8s-staging-cluster-api is k8s-infra-gcr-staging-cluster-api@googlegroups.com
+registries:
+  src: gcr.io/k8s-staging-cluster-api
+  dest: gcr.io/k8s-staging
+service-account: service-account@kubernetes-public.iam.gserviceaccount.com
+images:
```
Comment (listx) [inline k8s.gcr.io/k8s-staging-cluster-api/manifest.yaml]: You should do `images: []` to denote an empty list.

File: k8s.gcr.io/k8s-staging-csi/manifest.yaml (hunk 1 above)
Comment (listx) [reply k8s.gcr.io/k8s-staging-csi/manifest.yaml]: Err, might as well make that `images: []`.

File: k8s.gcr.io/k8s-staging-cluster-api/manifest.yaml (hunk 2 above)
Comment (dims) [reply k8s.gcr.io/k8s-staging-cluster-api/manifest.yaml]: ack

File: k8s.gcr.io/k8s-staging-csi/manifest.yaml (hunk 1 above)
Comment (dims) [reply k8s.gcr.io/k8s-staging-csi/manifest.yaml]: ack

**Comment by thockin** [comment]
I am impatient and need to send a PR on top of this

**Pull request merged**